src/metabase/api/search.clj

```diff
@@ -230,7 +230,9 @@
                                   visible-collections)
         honeysql-query           (-> honeysql-query
                                      (hh/merge-where collection-filter-clause)
-                                     (hh/merge-where [:= :collection.namespace nil]))]
+                                     (hh/merge-where [:or
+                                                      [:= :collection.namespace nil]
+                                                      [:= :collection.namespace "apps"]]))]
```

Comment:
This is for compatibility with the current version. Another option would be to make the frontend search in apps separately.

Reply:
One unified search endpoint makes sense, the last thing we would want is for the frontend client to be blasting the backend with multiple searches at the same time